crates/services/txpool/src/txpool.rs

```diff
@@ -71,41 +79,17 @@ where
         &self.by_dependency
     }
 
-    #[tracing::instrument(level = "info", skip_all, fields(tx_id = %tx.id(&self.config.chain_config.transaction_parameters.chain_id)), ret, err)]
+    pub fn current_height(&self) -> anyhow::Result<BlockHeight> {
+        Ok(self.database.current_block_height()?)
+    }
+
+    // #[tracing::instrument(level = "info", skip_all, fields(tx_id = %tx.id(&self.config.chain_config.transaction_parameters.chain_id)), ret, err)]
```

Comment:
Can we return it back?=)

Reply:
what to do with `id` field in the macro then since `CheckedTransaction` does not have it (directly)

Reply:
You can try to use `Checked<Transaction>` instead of `CheckedTransaction`. Then you can get transaction and get the id=)

Reply:
switched to `Checked<Transaction>`